### DISCUSSION TITLE
스크린샷 질문입니다.

### DISCUSSION THREAD

**Discussion by hipsc159** [naverz/zepeto-studio-kor#581] [5. Q&A]
![Screenshot_1](https://user-images.githubusercontent.com/68143858/161671013-9cd75c77-a9c5-4ef8-a305-14df23786e51.png)

빠진 스크립트나 파일은 없는거 같은데 이런 오류가 계속 뜨네요..
해결방법이 있을까요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 
ZepetoScreenShotModule의 인스펙터에 각 스크립트가 추가되어 있는지 확인 부탁드립니다.
<img width="350" alt="스크린샷 2022-04-05 오후 12 25 05" src="https://user-images.githubusercontent.com/100118650/161672812-db76f66a-53ba-49d6-990c-03ded90b74f7.png">

아래 깃헙 예제 기반에서 작업했을 때는 문제가 없는지도 확인 부탁드립니다.
https://github.com/naverz/zepeto-multiplay-example

**Reply by hipsc159** [reply]
네 깃헙예제에서는 잘 작동이 되었습니다.
아래의 사진처럼 전부 할당되어있고 그 자식들도 문제가 없는걸 확인했는데 이러네요 ㅠㅠ

![Screenshot_2](https://user-images.githubusercontent.com/68143858/161680327-3360efc7-b794-49d0-b1d3-da19242eaec6.png)

**Reply by pumfit** [reply]
해당 오류는 스크린샷 관련 스크립트를 모두 가져오지 않았을 경우 발생하는 것 같습니다.
자세한건 아래 링크 확인해보시면 좋을 것 같습니다!
https://github.com/naverz/zepeto-studio-kor/discussions/445

**Reply by hipsc159** [reply]
안그래도 비슷한 오류인줄 알고 확인해봤으나 스크립트 7개 모두 있었습니다..
스크립트 내부 코드 상의 문제일까요?

**Reply by Zepeto-TechEvangelist** [reply]
해당 이슈 확인해 보겠습니다.

**Reply by hipsc159** [reply]
넵 감사합니다!

**Reply by Zepeto-TechEvangelist** [reply]
확인 결과, 기존 프로젝트에 있는 스크립트 가져올 경우 제대로 인식하지 못해서 발생한 문제입니다.
탐색기에 들어가서 `meta파일을 삭제한 후` 다시 실행시키면 정상 동작 할 것입니다. 😄

**Reply by Zepeto-TechEvangelist** [reply]
이후에 다른 프로젝트에 적용된 스크립트를 옮길때는 메타 파일은 제외하고 본 파일만 옮겨주세요!

**Reply by hipsc159** [reply]
말씀하신 파일이
ZepetoScreenShotModule.prefab.meta 이 파일 맞나요?

**Reply by Zepeto-TechEvangelist** [reply]
넵 그 파일을 비롯하여 스크린샷 관련한 모든 파일들의 meta파일을 삭제하시기를 추천 드립니다.
